backend/status.go

```diff
@@ -0,0 +1,76 @@
+package backend
+
+import "net/http"
+
+type Status int
+
+const (
+	// StatusUnknown implies an error that should be updated to contain
+	// an accurate status code, as none has been provided.
+	// HTTP status code 500.
+	StatusUnknown Status = http.StatusInternalServerError
+
+	// StatusOK means that the action was successful.
+	// HTTP status code 200.
+	StatusOK Status = http.StatusOK
+
+	// StatusUnauthorized means that the server does not recognize the
```

Comment:
I wonder whether these should say `DataSource` instead of `Server` 🤔 we probably will be talking to a server of some kind but it looks _very_ HTTP-y right now